Lib/multiprocessing/resource_tracker.py

```diff
@@ -75,6 +75,20 @@ def _reentrant_call_error(self):
         raise ReentrantCallError(
             "Reentrant call into the multiprocessing resource tracker")
 
+    def __del__(self):
+        # making sure child processess are cleaned before ResourceTracker
+        # gets destructed.
+        # see https://github.com/python/cpython/issues/88887
+        try:
+            self._stop()
+        except AttributeError:
+            # AttributeError is likely caused by module teardown
+            # > __del__() can be executed during interpreter shutdown. As a
+            # > consequence, the global variables it needs to access (including
+            # > other modules) may already have been deleted or set to None.
+            # see https://docs.python.org/3/reference/datamodel.html#object.__del__
```

Comment:
You should fix the AttributeError rather than ignoring it silently. See my hack below.